lib/cryptoexchange/exchanges/bitstamp/services/pairs.rb

```diff
@@ -0,0 +1,26 @@
+module Cryptoexchange::Exchanges
+  module Bitstamp
+    module Services
+      class Pairs < Cryptoexchange::Services::Pairs
+        MARKET = Bitstamp::Market
```

Comment:
One possible improvement is to use the classname string and use that to find the yml